_pages/en_US/installing-boot9strap-(hbl-usm).txt

```diff
@@ -0,0 +1,86 @@
+---
+title: "Installing boot9strap (HBL-USM)"
```

Comment:
As mentioned above: I'd label this page "Installing boot9strap (slotTool)", personally, but I can see the consistency here

Reply:
as commented above